daemon/buildinfo.c

```diff
@@ -200,8 +200,67 @@
 
 #define FEAT_YES_NO(x) ((x) ? "YES" : "NO")
 
+// coverity[ +tainted_string_sanitize_content : arg-0 ]
+static inline void coverity_remove_taint(char *s)
+{
+    (void)s;
+}
+
+void read_install_type(void) {
+    char *filepath;
+    filepath = mallocz(sizeof(char) * (strlen(netdata_configured_user_config_dir) + strlen(".install-type") + 2));
+    sprintf(filepath, "%s/%s", netdata_configured_user_config_dir, ".install-type");
+    FILE *fp = fopen(filepath, "r");
+
+    if(fp) {
+        char *line = mallocz(sizeof(char) * (200 + 1));
```

Comment:
why the extra byte?
```
fgets() reads in at most one less than size characters from stream and stores them into the buffer pointed to by s. Reading stops after an EOF or a newline. If a newline is read, it is stored into the buffer. A terminating null byte (aq\0aq) is stored after the last character in the buffer.
```

Reply:
The terminating null byte that should be at the end of the string. Any buffer that’s having a string read into it needs to be at least one byte longer than the longest string it is expected to contain.

Reply:
but fgets will ensure null byte is within the limit